packages/mds-metrics/utils.ts

```diff
@@ -8,9 +9,7 @@ export function getTimeBins({
 }: Partial<GetTimeBinsParams>) {
   const interval = end_time - start_time
 
-  const bins = new Array(Math.floor(interval / bin_size))
-
-  return bins.map((_, idx) => ({
+  return [...Array(Math.floor(interval / bin_size))].map((_, idx) => ({
```

Comment:
Do we need to `...` spread this, or can we just call `Array().map()`?

Reply:
Need to spread it. If you don't, it creates an array of the specified length, but doesn't actually have any items. So, when you try to call `.map()` on it, nothin' happens!

Reply:
Another alternative is to use `Array.from` which supports a map function over an _array-like_ object, i.e.
```
Array.from({ length: Math.floor(interval / bin_size) }, (_,idx) => { ... })
```

Reply:
I have no strong opinion either way, I just wanted to understand what it was doing

Reply:
Me either, but it seems `Array.from` was designed for this sort of use case.